packages/@angular/cli/lib/init.ts

```diff
@@ -91,7 +91,7 @@ resolve('@angular/cli', { basedir: process.cwd() },
         Your global Angular CLI version (${globalVersion}) is greater than your local
         version (${localVersion}). The local Angular CLI version is used.
 
-        To disable this warning use "ng set --global warnings.versionMismatch=false".
+        To disable this warning use "ng config -g cli.warnings.typescriptMismatch false".
```

Comment:
Not sure if this is the correct command to disable the warning

Reply:
This should be `cli.warnings.versionMismatch`.   There's only two controllable warnings currently: https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/lib/config/schema.json#L53